game-core/src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -141,53 +138,6 @@ public static synchronized boolean headless() {
     return availableGames.getGameNames();
   }
 
-  public synchronized void setGameMapTo(final String gameName) {
```

Comment:
I was assuming this is a leftover from UI code from bots.
I'll have a look though, somewhere in the netcode should be a method doing something like this.

Reply:
I could track down the original implementation to https://github.com/triplea-game/triplea/commit/13d4df82497994714e15c7bff7b58c7d66c470fa#diff-b5bd9d9c797f9ac8e81743d0b7c15832
Where the interface `IClientMessenger` is created, and the HeadlessGameServer class implements it.
However somewhere along history, this HeadlessGameServer was no longer implementing the interface, but the method stayed.
You can find the remaining implementation in `ClientMessenger.java`

Reply:
UPDATE: Looking again I noticed a mistake with my theory.
It's while it's true that the setGameMapTo method inside HeadlessGameServer is currently unused and not invoked via reflection, the only remaining usage was removed from ServerModel, which _was_ being invoked reflectively by the ClientMessenger (which I didn't expect, because usually we create a remote instance of an interface and then work with the interface.).

So good catch. Reverting.